plugins/events-manager/templates/events-search.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/* WARNING!!! (2013-07-10) We intend to add a few more fields into this search form over the coming weeks/months. 
+ * Overriding shouldn't hinder functionality at all but these new search options won't appear on your form! 
+ */ 
+/* 
+ * By modifying this in your theme folder within plugins/events-manager/templates/events-search.php, you can change the way the search form will look.
+ * To ensure compatability, it is recommended you maintain class, id and form name attributes, unless you now what you're doing. 
+ * You also have an $args array available to you with search options passed on by your EM settings or shortcode
+ */
+$args = !empty($args) ? $args:array(); /* @var $args array */
+?>
+<div class="em-search-wrapper row">
+<div class="em-events-search em-search col-lg-7 col-sm-12 <?php if( !empty($args['main_classes']) ) echo esc_attr(implode(' ', $args['main_classes'])); ?>">
+  <?php $view = get_query_var( 'view', $default = ''); ?>
+
+  <?php $args['search_url'] = 'http://mdmozdev.wpengine.com.test/events/?view='.$view ?>
```

Comment:
We should not have hardcoded URLs in the code.  You could use $_SERVER['HTTP_HOST']